src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -602,11 +601,12 @@ The [Cognito Hosted UI]( https://docs.aws.amazon.com/cognito/latest/developergui
 After you test the sign-up, sign-in, and sign-out functionality, you can take a look at the created user in the Amazon Cognito console. To do so, you can use `amplify console auth` to open the console directly from the CLI, or take a look at the existing User Pools in the console. Note the users status should be "CONFIRMED" if you successfully verified the user; however, it can be in "unconfirmed" if the user is not yet verified. You can also check to make sure any other required attributes are updating correctly.
 
 **Recap:** You now have enabled sign-up, sign-in, and sign-out for your authentication and verified its functionality with a test user.
-
+ 
```

Comment:
For the Recap above - is there anything more we want to add on to tie-in to real-world use and application of the topic discussed?

Reply:
No - it is fine as is for now. We can re-look at it after we make some of the changes we discussed.